code/modules/events/solar_storm.dm

```diff
@@ -39,7 +41,7 @@
 			continue
 
 		//Apply some heat or burn damage from the sun.
-		if(istype(L, /mob/living/carbon/human))
+		if(istype(L, /mob/living/carbon/human) || istype(L, /mob/living/exosuit))
```

Comment:
This should be some `AdjustBodyTemp()` proc that humans and exosuits can override.

Reply:
That naming was used by a proc that I dont quite understand, but done.